client/src/components/DefaultButton.js

```diff
@@ -0,0 +1,26 @@
+import styled from 'styled-components';
```

Comment:
```suggestion
import styled from 'styled-components/macro';
```

See: https://styled-components.com/docs/tooling#babel-macro